control-plane-operator/controllers/hostedcontrolplane/oapi/deployment.go

```diff
@@ -128,6 +128,7 @@ func ReconcileDeployment(deployment *appsv1.Deployment, ownerRef config.OwnerRef
 				}
 			}),
 		},
+		TerminationGracePeriodSeconds: pointer.Int64Ptr(90),
```

Comment:
Can you please add go docs for both of them why this is needed like here https://github.com/openshift/hypershift/blob/eb89a6ce1c655d3eba0b80a2884198c7488505fb/control-plane-operator/controllers/hostedcontrolplane/kas/deployment.go#L137-L140?

Please link to something more meaningful than "This is what roks does"